src/main/dynamo/util.ts

```diff
@@ -28,6 +29,26 @@ export async function decryptOrPassThroughItem(
     await jayz.ready
     return jayz.decryptItem(item as EncryptedJayZItem<any, any>)
   } else {
-    return item
+    return formatDynamoDBItem(item)
   }
 }
+
+/**
+ * The DynamoDB v3 lib returns items with values that are UInt8Array instead of Buffer, so this function
```

Comment:
The response format appears to have changed from Buffer to UInt8Array, so we're converting to keep the expected format for clients.